frontend/src/views/camp/CampProgram.vue

```diff
@@ -45,13 +56,19 @@ export default {
     Picasso,
     ScheduleEntries
   },
+  mixins: [campRoleMixin],
   props: {
     period: { type: Function, required: true }
   },
   data () {
     return {
       editMode: false
     }
+  },
+  computed: {
+    camp () {
+      return this.period().camp()
+    }
```

Comment:
@usu genau aus diesem Grund wäre wohl das CampRoleMixin besser ein composable, oder? Dann wäre garantiert, dass es ein Camp hat.

Reply:
Ja, genau. Dann würdest du das Camp explizit als Parameter an das Composable übergeben.

Mit dem Mixin hat's halt diese explizite Interface-Definition nicht: Du musst selbst wissen, dass das Mixin `this.camp` erwartet (bzw. über trial und error herausfinden).